### ISSUE TITLE
Can you give me your data

### ISSUE THREAD

**Issue by wenjiechen0913** [wglassly/cnnormaliztion#2]
I would like to directly reproduce your code model and find that there is a lack of original data. Could you please give me the data of your training set, test set and verification set so that I can successfully run the model and learn from it

**Comment by meghbhalerao** [comment]
Hi @WjieChen,
Do you have the readme file for this or could you let me know which is the starting point for this?
Thanks,
Megh

**Comment by meghbhalerao** [comment]
Hi @WjieChen - do you know how to generate the candidate.txt file?

I can provide you with the training and test set if you need it. It is here  - https://github.com/jennydsuza9/disorder-normalizer/tree/b48bf4f7a03a5f1deb6bf9d4e9585387265e45af/ncbi-data
